source/unified-test-format/unified-test-format.rst

```diff
@@ -1779,6 +1787,54 @@ The following arguments are supported:
 
 - ``session``: Optional string. See `commonOptions_session`_.
 
+runCursorCommand
+~~~~~~~~~~~~~~~~
+
+`Generic cursor returning command runner <../run-command/run-command.rst>`__.
+
+This method does not inherit a read concern or write concern (per the
+`Read and Write Concern <../read-write-concern/read-write-concern.rst#generic-command-method>`__
+spec), nor does it inherit a read preference (per the
+`Server Selection <../server-selection/server-selection.rst#use-of-read-preferences-with-commands>`__
+spec); however, they may be specified as arguments.
+
+This operation proxies the databases's ``runCursorCommand`` method and supports the same arguments and options.
+
+When executing the provided command, the test runner MUST fully iterate the cursor.
+This will ensure consistent behavior between drivers that eagerly create a server-side cursor and those that do so lazily when iteration begins.
+
+The following arguments are supported:
+
+- ``command``: Required document. The command to be executed.
+
+- ``commandName``: Required string. The name of the command to run. This is used
+  by languages that are unable preserve the order of keys in the ``command``
+  argument when parsing YAML/JSON.
+
+- ``readPreference``: Optional object. See `commonOptions_readPreference`_.
+
+- ``session``: Optional string. See `commonOptions_session`_.
+
+- ``batchSize``: Optional positive integer value. Use this value to configure the ``batchSize`` option sent on subsequent ``getMore`` commands.
+
+- ``maxTimeMS``: Optional positive integer value. Use this value to configure the ``maxTimeMS`` option sent on subsequent ``getMore`` commands.
+
+- ``comment``: Optional BSON value. Use this value to configure the ``comment`` option sent on subsequent ``getMore`` commands.
+
+- ``cursorType``: Optional string enum value, one of ``'tailable' | 'tailableAwait' | 'nonTailable'``. Use this value to configure the enum passed to the ``cursorType`` option.
+
+- ``timeoutMode``: Optional string enum value, one of ``'iteration' | 'cursorLifetime'``. Use this value to configure the enum passed to the ``timeoutMode`` option.
+
+createCommandCursor
+~~~~~~~~~~~~~~~~~~~
+
+This operation proxies the database's ``runCursorCommand`` method and supports the same arguments and options.
+Test runners MUST ensure that the server-side cursor is created (i.e. the command document has executed) as part of this operation and before the resulting cursor might be saved with `operation.saveResultAsEntity <operation_saveResultAsEntity_>`_.
+Test runners for drivers that lazily execute the ``runCursorCommand`` command on the first iteration of the cursor MUST iterate the resulting cursor once.
```

Comment:
```suggestion
Test runners for drivers that lazily execute the command on the first iteration of the cursor MUST iterate the resulting cursor once.
```

The monospace formatting for `runCursorCommand` looks like copypasta from `createFindCursor`, which referred to the `find` command explicitly. Here, the command can be one of various things.